lua/lazyvim/plugins/extras/editor/overseer.lua

```diff
@@ -0,0 +1,103 @@
+return {
+  {
+    "catppuccin",
+    optional = true,
+    opts = {
+      integrations = { overseer = true },
+    },
+  },
+  {
+    "folke/which-key.nvim",
+    optional = true,
+    opts = {
+      defaults = {
+        ["<leader>o"] = { name = "+overseer" },
+      },
+    },
+  },
+  {
+    "stevearc/overseer.nvim",
+    cmd = {
+      "OverseerOpen",
+      "OverseerClose",
+      "OverseerToggle",
+      "OverseerSaveBundle",
+      "OverseerLoadBundle",
+      "OverseerDeleteBundle",
+      "OverseerRunCmd",
+      "OverseerRun",
+      "OverseerInfo",
+      "OverseerBuild",
+      "OverseerQuickAction",
+      "OverseerTaskAction",
+      "OverseerClearCache",
+    },
+    {
+      "folke/edgy.nvim",
+      optional = true,
+      opts = function(_, opts)
+        opts.right = opts.right or {}
+        table.insert(opts.right, {
+          title = "Overseer",
+          ft = "OverseerList",
+          open = function()
+            require("overseer").open()
+          end,
+        })
+      end,
+    },
+    opts = {
+      dap = false,
+      task_list = {
+        bindings = {
+          ["<C-h>"] = false,
+          ["<C-j>"] = false,
+          ["<C-k>"] = false,
+          ["<C-l>"] = false,
+        },
+      },
+      form = {
+        win_opts = {
+          winblend = 0,
+        },
+      },
+      confirm = {
+        win_opts = {
+          winblend = 0,
+        },
+      },
+      task_win = {
+        win_opts = {
+          winblend = 0,
+        },
+      },
+    },
+    -- stylua: ignore
+    keys = {
+      { "<leader>ow", "<cmd>OverseerToggle<cr>",      desc = "Task list" },
+      { "<leader>oo", "<cmd>OverseerRun<cr>",         desc = "Run task" },
+      { "<leader>oq", "<cmd>OverseerQuickAction<cr>", desc = "Action recent task" },
+      { "<leader>oi", "<cmd>OverseerInfo<cr>",        desc = "Overseer Info" },
+      { "<leader>ob", "<cmd>OverseerBuild<cr>",       desc = "Task builder" },
+      { "<leader>ot", "<cmd>OverseerTaskAction<cr>",  desc = "Task action" },
+      { "<leader>oc", "<cmd>OverseerClearCache<cr>",  desc = "Clear cache" },
+    },
+  },
+  {
+    "nvim-neotest/neotest",
+    optional = true,
+    dependencies = "stevearc/overseer.nvim",
+    opts = {
+      consumers = {
+        overseer = require("neotest.consumers.overseer"),
```

Comment:
This will always load neotest & overseer during spec parsing. 
Please put it in a closure

Reply:
This?
```lua
{
  "nvim-neotest/neotest",
  optional = true,
  opts = function(_, opts)
    opts = opts or {}
    opts.consumers = opts.consumers or {}
    opts.consumers.overseer = require("neotest.consumers.overseer")
  end,
}
```

Reply:
Oh, I know.